### DISCUSSION TITLE
Contact message fails on mail server, needs a configurable "noreply"

### DISCUSSION THREAD

**Discussion by skrey** [datenstrom/community#786] [Report a bug] [answered]
Hi.

I have added a configuration option to set the from address used by the contact extension as the default "noreply" is not accepted from the used mail server.

I have created a pull request for inclusion into the yellow-extensions repo.

Best regards,
Sebastian

**Comment by schulle4u** [comment, marked as answer]
Merged and published, thank you! 👍

The contact and edit extensions now support a custom from address, configurable via the `ContactSiteEmail` and `EditSiteEmail` settings respectively. This should solve email sending issues, please let us know if you still have problems.
